react/components/SideBar.tsx

```diff
@@ -15,9 +15,17 @@ interface Chapter {
   articles: [] | Chapter[]
 }
 
+function pageView() {
+  if (typeof ga() === 'undefined') {
+    initialize('UA-150301985-1')
+  }
+  pageview(window.location.pathname + window.location.search)
+}
+
 const SideBar: FC = () => {
   const appName = 'vtex.io-documentation@0.x'
   const { content } = useSideBarContentState()
+  useEffect(() => pageView())
```

Comment:
You could add an empty dependency array to this `useEffect` call, just to be sure this will only be triggered when the component mounts, kinda like `componentDidMount`.
Also, it's a good practice to always pass a dependency array to `useEffect` so that React optimizes it.

Reply:
But the idea is to mimic the effect of `componentDidUpdate`. So I think not passing a dependency array is the desired use.

Reply:
I was thinking about creating an `url` state variable and passing it to the dependency array, to ensure that the `useEffect` is only called when the `url` variable is updated. But I could not create a way to avoid an infinite loop. What do you think?

Reply:
Oooh, I see. Then I guess it's fine.
And the state variable would not work because it would trigger a rerender to set it... I think it's fine the way it is :)

Reply:
Great!